src/components/atoms/Heading/Heading.tsx

```diff
@@ -26,8 +32,10 @@ interface IStyledHeadingProps {
 const Heading = styled.h1<IStyledHeadingProps>`
   color: ${({ color = colors.neutral.dark }) => color};
   
-  font-size: ${({ as }) => headingSizes[as]};
+  font-size: ${({ as, size }) => (as === 'span' || size ? headingSizes[size || 'h1'] : headingSizes[as])};
```

Comment:
defaulting to `h1` size?

Reply:
Any other preference, maybe we should default to `h4` or `h6` instead? 💭

Reply:
I think I'd rather make `size` prop required and default `as` to `size` if it's not provided. Breaking change but probably a bit more elegant

Reply:
I see having a different twist 🤔 , I guess is more elegant as you say however it might tricky to update for clients ( _given `as` is also used in `<Text />` hence find and replace will end messy_ )

Reply:
I default it to **h4 size**  ( _which is less aggressive visually_ ) for now 💄 👍🏻

I think given there's also the icons update I prefer to keep this as not a _breaking change_, for now, to remove upgrade hassle on the clients 🙈